testsuite/MDAnalysisTests/visualization/test_streamlines.py

```diff
@@ -67,22 +67,10 @@ def test_streamplot_2D(membrane_xtc, univ, tmpdir):
                                                         ymax=univ.atoms.positions[...,1].max(),
                                                         maximum_delta_magnitude=2.0,
                                                         num_cores=1)
-    x = np.linspace(univ.atoms.positions[...,0].min(),
-                    univ.atoms.positions[...,0].max(),
-                    5)
-    y = np.linspace(univ.atoms.positions[...,1].min(),
-                    univ.atoms.positions[...,1].max(),
-                    5)
-    speed = np.sqrt(u1*u1 + v1*v1)
-    fig = plt.figure()
-    ax = fig.add_subplot(111, aspect='equal')
-    ax.streamplot(x, y, u1, v1, density=(10,10), color=speed, linewidth=3*speed/speed.max(),
-                  cmap=plt.cm.viridis)
-    plot_outpath = os.path.join(str(tmpdir), 'testing_streamline.png')
-    fig.savefig(plot_outpath, dpi=300)
-
-    with open(plot_outpath, 'rb'):
-        pass
+    assert u1.shape == (5, 5)
+    assert v1.shape == (5, 5)
+    assert_allclose(avg, 0.965194167)
```

Comment:
Are these single value checks? If so could we use pytest.approx instead (it is much faster last I checked)?